pkg/cmd/flags/capture.go

```diff
@@ -18,20 +18,21 @@ Possible options:
 [artifact:]exec                    capture executed files.
 [artifact:]module                  capture loaded kernel modules.
 [artifact:]mem                     capture memory regions that had write+execute (w+x) protection, and then changed to execute (x) only.
-[artifact:]net=interface           capture network traffic of the given interface. Only TCP/UDP/ICMP protocols are currently supported.
+[artifact:]network                 capture network traffic. Only TCP/UDP/ICMP protocols are currently supported.
 
 dir:/path/to/dir                    path where tracee will save produced artifacts. the artifact will be saved into an 'out' subdirectory. (default: /tmp/tracee).
 profile                             creates a runtime profile of program executions and their metadata for forensics use.
 clear-dir                           clear the captured artifacts output dir before starting (default: false).
-pcap:[per-container|per-process]    capture separate pcap file based on container/process context (default: none - saving one pcap for the entire host).
+netpcap:[process,container,command] capture separate pcap files organized by processes, containers and/or commands
```

Comment:
https://github.com/aquasecurity/tracee/pull/2547/files#r1067386220

Reply:
also change documentation when this is done (http://localhost:8000/tracee/docs/capturing/)